qa/L0_tf_unknown_rank/test.sh

```diff
@@ -101,6 +98,25 @@ set -e
 kill $SERVER_PID
 wait $SERVER_PID
 
+# Try to load model with scalar tensor. The server should fail to load the model.
+rm -rf scalar_repo; mkdir scalar_repo
+cp -r $DATADIR/tf_model_store2/scalar_model scalar_repo/
```

Comment:
All models in tf_model_store2 must load correct due to other CI tests. scalar_model will need to live in tf_model_store3 or similar new repo.